src/main/java/mate/academy/Main.java

```diff
@@ -2,16 +2,39 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import mate.academy.exception.AuthenticationException;
+import mate.academy.exception.RegistrationException;
+import mate.academy.lib.Injector;
 import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
 import mate.academy.model.MovieSession;
+import mate.academy.model.User;
+import mate.academy.security.AuthenticationService;
 import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
 import mate.academy.service.MovieSessionService;
 
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
-        MovieService movieService = null;
+        AuthenticationService authenticationService = (AuthenticationService) injector
+                .getInstance(AuthenticationService.class);
+
+        try {
+            authenticationService.register("guf@dgigurda.com", "1234");
+        } catch (RegistrationException e) {
+            throw new RuntimeException("Authentication failed", e);
+        }
```

Comment:
exception will be thrown if it occurs from AuthService, and in those implementation catch block do the same and program will fail with Registration exception.
IMHO: we dont need try/catch here